dlt-connector/src/graphql/resolver/TransactionsResolver.ts

```diff
@@ -48,6 +50,7 @@ export class TransactionResolver {
         // we can store the transaction and with that automatic the backend transaction
         await transactionRecipe.save()
       }
+      InterruptiveSleepManager.getInstance().interrupt(TRANSMIT_TO_IOTA_INTERRUPTIVE_SLEEP_KEY)

```

Comment:
in tasks/transmitToIota.ts:
```ts
await InterruptiveSleepManager.getInstance().sleep(
        TRANSMIT_TO_IOTA_INTERRUPTIVE_SLEEP_KEY,
        1000,
      )
```
It will interrupt the sleep of the transmitToIota Loop.
It is modeled after conditions used in multithreading in java and c++
The Loop in transmitToIota:
- check for new transactions for sending to iota
- sleep 10000 ms 
- check again
- if  InterruptiveSleepManager.getInstance().interrupt(TRANSMIT_TO_IOTA_INTERRUPTIVE_SLEEP_KEY) is called, while sleep, the sleep will be interrupted and the loop check for new transactions to sending to iota.